### DISCUSSION TITLE
I am LOST, please help me

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22442] [New to GitHub]
<p>I found the exceptional AUDACITY program on the web. Great. But, the infinity of resources, parameters, details, often makes me get lost and have to start over, but I’m learning. I ask new friends, if possible, to post help in PORTUGUESE, as the translator often translates wrongly.I was directed to GITHUB, made the account, now I was directed to this forum. I’m lost, laughs. I want help with AUDACITY parameters and secrets.</p>

**Comment by airtower-luna** [comment]
<p>For trouble with a specific software that’s hosted on GitHub (as Audacity is) it’s usually best to create an issue or discussion <em>in that software’s repository</em>, unless the README lists other support channels. This forum is primarily about Git and GitHub itself, so it’s usually not the best place to ask about issues with other software.</p>
<p>The Audacity repository is here:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/audacity/audacity">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/audacity/audacity" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/5814919/181070933-92e25b0a-d550-4cc5-89e1-8ffe0f842642.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e8c096ad4163dc6547f0a71c982ad5a2067b7bbc_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e8c096ad4163dc6547f0a71c982ad5a2067b7bbc_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/8/e8c096ad4163dc6547f0a71c982ad5a2067b7bbc.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e8c096ad4163dc6547f0a71c982ad5a2067b7bbc_2_10x10.png"></div>

<h3><a href="https://github.com/audacity/audacity" target="_blank" rel="noopener">GitHub - audacity/audacity: Audio Editor</a></h3>

  <p>Audio Editor                                     . Contribute to audacity/audacity development by creating an account on GitHub.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>I noticed the README mentions a user forum, but they also seem to use issues and discussions.</p>
<p>Good luck! :slightly_smiling_face:</p>

**Comment by mattwelke** [comment, marked as answer]
<p>Just wanted to add that I see Audacity has enabled Discussions too:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/audacity/audacity/discussions">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/audacity/audacity/discussions" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/7719209/181070935-24e05e40-52b8-4003-bbde-8a6955e84ff4.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e8c096ad4163dc6547f0a71c982ad5a2067b7bbc_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e8c096ad4163dc6547f0a71c982ad5a2067b7bbc_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/8/e8c096ad4163dc6547f0a71c982ad5a2067b7bbc.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e8c096ad4163dc6547f0a71c982ad5a2067b7bbc_2_10x10.png"></div>

<h3><a href="https://github.com/audacity/audacity/discussions" target="_blank" rel="noopener nofollow ugc">Discussions · audacity/audacity</a></h3>

  <p>Audio Editor                                     . Contribute to audacity/audacity development by creating an account on GitHub.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>That’s usually a better place to ask questions to get help than creating an issue if the repository owner has enabled it.</p>

**Comment by ghost** [comment]
<p>Thank you, ainda preciso aprender muito.</p>
